sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -515,6 +613,65 @@ def test_one_job_fails_all_jobs_fail(self):
 
     hamcrest_assert(p, all_of(*pipeline_verifiers))
 
+  @unittest.skip('TODO(BEAM-6611) - BQ file loads in streaming')
+  @attr('IT')
+  def test_one_job_fails_all_jobs_fail_streaming(self):
```

Comment:
For a streaming pipeline, it will never stop running, right? It will continue producing errors, and continue retrying a bundle until it's stopped. We'd need to find a way to stop the pipeline.

Reply:
Oh yes, I hadn't thought of that. Thank You :)
I'll think of a way.

Reply:
We could avoid doing `with Pipeline(..) as p:`, because this calls `wait_until_finish`. Instead, use `p = Pipeline(..)`, and `p.run()`, and stop the job after a while. 
But I think we could leave that as a TODO for after implementation - and for now just work on the test case that it works.
Thoughts?

Reply:
So sorry, @pabloem I just saw this :(
In the new PR for this, I made a change to the `TestDirectRunner` so that `wait_until_finish` is called for streaming pipelines[1]. Since it was not being called earlier, the on_success_matcher was being matched against results from the first run of the streaming pipeline. This was causing the errors. 

I agree. I'll add it after the implementation is locked down. I'll create a JIRA for it.

[1] https://github.com/ttanay/beam/blob/2f510bf6fc5cd77ea106b467cb3939b57dc100fa/sdks/python/apache_beam/runners/direct/test_direct_runner.py#L48